modules/storage-api/src/test/java/org/apache/ignite/internal/storage/index/impl/PrefixComparator.java

```diff
@@ -15,77 +15,67 @@
  * limitations under the License.
  */
 
-package org.apache.ignite.internal.storage.index;
+package org.apache.ignite.internal.storage.index.impl;
 
 import java.util.Arrays;
 import java.util.BitSet;
-import org.apache.ignite.internal.schema.BinaryRow;
-import org.apache.ignite.internal.schema.Column;
-import org.apache.ignite.internal.schema.NativeTypeSpec;
-import org.apache.ignite.internal.schema.row.Row;
+import org.apache.ignite.internal.schema.BinaryTuple;
+import org.apache.ignite.internal.schema.NativeType;
+import org.apache.ignite.internal.storage.index.IndexRowPrefix;
+import org.apache.ignite.internal.storage.index.SortedIndexDescriptor;
 import org.apache.ignite.internal.storage.index.SortedIndexDescriptor.ColumnDescriptor;
 import org.jetbrains.annotations.Nullable;
 
 /**
- * Class for comparing a {@link BinaryRow} representing an Index Key with a given prefix of index columns.
+ * Class for comparing a {@link BinaryTuple} representing an Index Row with a given prefix of index columns.
  */
-public class PrefixComparator {
+class PrefixComparator {
```

Comment:
I expect this class to disappear in the future, is this a reasonable expectation? There should be a single place with comparison implementation

Reply:
yes, I left it for simplicity purposes

Reply:
I removed this class